portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase

```diff
@@ -761,6 +761,41 @@ definition {
 		WorkflowMetrics.validateInstanceDetails(processStatus = "Completed");
 	}
 
+	@description = "LRQA-53659 Automate LPS-86652 [Reports] View all items for a certain process"
+	test ViewReports {
+		property portal.acceptance = "false";
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Site Administration",
+			portlet = "Workflow");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry2 Content",
+			entryTitle = "Blogs Entry2 Title");
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry1 Content",
+			entryTitle = "Blogs Entry Title");
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		Click(locator1 = "WorkflowMetrics#DASHBOARD_TOTAL_PENDING_ITEMS_All_ITEMS");
```

Comment:
hi @Adriverforlife ,
Is DASHBOARD_TOTAL_PENDING_ITEMS_VALUE not applicable for this case?

Reply:
You are right. In my manual testing I missed that you could click anywhere in that box.I swapped this around and got rid of the unneeded path.